restord/ImageSessionServer.m

```diff
@@ -444,4 +450,13 @@ - (void)applyImageInfo:(NSURL *)mountURL {
   }
 }
 
+- (void)blessMountURL:(NSURL *)mountURL {
+  NSTask *bless = [[NSTask alloc] init];
+  bless.launchPath = @"/usr/sbin/bless";
+  bless.arguments = @[ @"--folder", mountURL.path ];
+  [bless launch];
+  [bless waitUntilExit];
+  if (bless.terminationStatus != 0) NSLog(@"%@ Failed to bless: %@", self, mountURL.path);
```

Comment:
Does this fail on older versions of bless? Older versions of macOS?

Reply:
probably - requiring 10.13.4